lib/render/editor/editor.html

```diff
@@ -497,9 +508,46 @@
 
       _onExecutionStop: function (data) {
         this._cleanState();
+        this._setResultsBoards(data.test_results);
         mumuki.kids.showResult(data);
       },
 
+      _setResultsBoards: function (testResults) {
+        if (!testResults) return;
+        let testResultsBoards = this._testResultsBoards(testResults);
+        this._updateBoards(testResultsBoards);
+        let testResultsStatuses = this._testResultsStatuses(testResults);
+        this.multipleScenarios.updateIndicators(testResultsStatuses);
+      },
+
+      _testResultsBoards: function (testResults) {
+        return testResults.map(it => $(it.result).find('.actual_board > *, .final_board > *')[0]);
+      },
+
+      _testResultsStatuses: function (testResults) {
+        return testResults.map(it => it.status);
+      },
+
+      _updateBoards: function (resultsBoards) {
+        let initialBoards = this.multipleScenarios.scenarios[0];
+        resultsBoards.forEach((resultScenario, index) => {
+          if (this.multipleScenarios.currentScenarioIndex === index) return;
+          let currentBoard = initialBoards[index];
```

Comment:
This variable is not being used

Reply:
currentBoard? It's being used in the next line. It should be a const though :+1:

Reply:
My bad! :stuck_out_tongue: